core/src/components/loading/loading.md.scss

```diff
@@ -35,9 +35,9 @@
 // Material Design Loading Spinner fill colors
 // -----------------------------------------
 
-.loading-md .spinner-ios line,
-.loading-md .spinner-ios-small line {
```

Comment:
This was the name of the actual spinner, but it was renamed to `lines`. Can you revert all changes to spinners and make a separate issue for it? These need to be gone through one by one.